x-pack/plugins/security_solution/public/resolver/store/data/action.ts

```diff
@@ -53,9 +57,32 @@ interface ServerReturnedRelatedEventData {
   readonly payload: ResolverRelatedEvents;
 }
 
+interface ServerReturnedRelatedEventsOfType {
+  readonly type: 'serverReturnedRelatedEventsOfType';
+  readonly payload: {
```

Comment:
Not sure if it makes sense, but could mark the type of `payload` as `NodeEventsInCategoryState` since they're the same?

Reply:
👍